lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -452,7 +452,7 @@ sub prepare_tfvars_file {
     assert_script_run($retrieve_tfvars_cmd);
     assert_script_run("test -f $tfvars_file");
     replace_tfvars_variables($tfvars_file);
-    upload_logs($tfvars_file, log_name => "$args{deployment_type}.tfvars");
+    upload_logs($tfvars_file, log_name => "$args{deployment_type}.tfvars.txt");
```

Comment:
in other places (qesapdeployment.pm) I decided not to do it to keep the specific file extension. But it is up to you.

Reply:
Let's keep it for now how will it turn out. In the worst case I will change it.
The reasoning behind this is that in case of SDAF bug I will be sending tfvars to MS. 
It might be less confusing if it has original extension as well and not only txt.